.env.example

```diff
@@ -1,5 +1,6 @@
 NUXT_PUBLIC_SENTRY_DSN=
 NUXT_PUBLIC_APP_DOMAIN=localhost:8080
+NUXT_PUBLIC_PLAUSIBLE_DOMAIN=localhost:8080
```

Comment:
.env.example is for running the app on localhost. In non-localhost environment the user needs to set an appropriate value just like in all other cases.

NUXT_PUBLIC_PLAUSIBLE_DOMAIN is the name of the env variable that the plugin expects to be configured if we want to resolve that value in the runtime. docs: https://github.com/nuxt-modules/plausible#runtime-config

Similar case with `siteUrl` config that sitemap plugin expects 

> Thank you! This can only be checked after merging, right?
I'm afraid so :( we don't use plausible on preview environments

Reply:
Got it. Thanks for explanation